App/PHP_HTML/profile.php

```diff
@@ -84,68 +95,85 @@ function openTab(evt, cityName) {
         </tr>
     </table>
     </br>
+	
 
     <!--Reference: https://www.w3schools.com/howto/howto_css_icon_bar.asp -->
+	<!-- This tab bar lets you navigate to view user's information and activity  -->
     <div class="tab">
-        <button class="tablinks" onclick="openTab(event, 'about')"><a class="active" href="#"><i
-                        class="fa fa-address-card"></i></a></button>
-        <button class="tablinks" onclick="openTab(event, 'mail')"><a href="#"><i class="fa fa-envelope"></i></a>
-        </button>
-        <button class="tablinks" onclick="openTab(event, 'activity')"><a href="#"><i class="fa fa-line-chart"></i></a>
+		<!--Opens the activity tab -->
+        <button class="tablinks" onclick="openTab(event, 'activity')"><a class="active" href="#"><i class="fa fa-line-chart"></i></a>
         </button>
+		<!--Opens the question tab which displays all the questions the user asked -->
+		<button class="tablinks" onclick="openTab(event, 'question_mark')"><a href="#"><i class="fa fa-question"></i></a>
+		</button>
+		<!-- Opens the edit profile tab-->
         <button class="tablinks" onclick="openTab(event, 'edit')"><a href="#"><i class="fa fa-edit"></i></a></button>
     </div>
-
+	
+	<!-- Reference: https://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_progressbar3&stacked=h -->
+	<!-- Added a progress bar to illustrate user's activity and personal information -->
     <div id="activity" class="tabcontent">
         <table class="t2">
             <tr>
-                <th><h3>Activity</h3></th>
+                <th><h2> User's Activity : </h2></th>
             </tr>
             <tr>
                 <td>
-                    <ul class="">
-                        <li class=""><strong class="">Questions asked: <?php echo $user['user_questions_count']; ?></strong></li>
-                        <li class=""><strong class="">Answers: <?php echo $user['user_answers_count']; ?></strong></li>
-                        <li class=""><strong class="">Likes</strong></li>
-                        <li class=""><strong class="">Followers</strong></li>
-                    </ul>
+				<ul>
+					<div class="">

```

Comment:
any point to empty class name?

Reply:
not anymore... had plans to add a class to deal with user subs, but gave up on them :p